### ISSUE TITLE
Accounts for Signer endpoint in Horizon using new ingestion system

### ISSUE THREAD

**Issue by bartekn** [stellar/go#1472]
We're going to test new `exp/ingest` package by creating a simple "Accounts for Signer" endpoint in Horizon (#432). The new ingestion system will be completely separated from an existing ingestion of Horizon. There will be no synchronization so it's possible that "Accounts for Signer" data will not be presenting data at the same ledger as other Horizon endpoints.

Here's a proposed list of steps to achieve this is below. Each step will be moved to another issue once the plan is approved.

- [x] #1459 We need to find a way to report status updates of a session in the Horizon logs.
- [x] #1505 Remove concurrency feature from Pipeline.
- [x] #1311 We need to ensure that using malicious history archives is not a security risk. I.e. is presenting invalid "Account for Signers" can cause any security risk or it's just inconvenience?
- [x] #1499 `LiveSession` should update cursor value in Horizon
- [x] https://github.com/stellar/go/issues/1473 Sessions should continue on transient errors
- [x] https://github.com/stellar/go/issues/1484 Add new configuration parameters to Horizon. We need history archives URL to initialize the ledger state.
- [x] https://github.com/stellar/go/issues/1485 Create database migration for a new table with signers and all necessary methods to add/remove rows.
- [x] https://github.com/stellar/go/issues/1486 Create new action presenting data from a database.
- [x] https://github.com/stellar/go/issues/1487 Create a feature flag for this feature (default=off).
- [x] Use `LiveSession` (in #1456) and create pipelines that pull data from history archives and ledger backend and put them in a database. This is mostly done in `horizon-demo` in #1456.
- [x] Add tests for a new feature. Monitor it when RC is released to testnet.

@ire-and-curses @tamirms would you like to add/modify something?

**Comment by ire-and-curses** [comment]
I think it looks good. Do you think the in-memory reader will be sufficient for this, or do we also need to look at #1457? Maybe the tests @tamirms is working on will help us decide.

How will we test the final endpoint? Might be worth a card for that as well.

**Comment by bartekn** [comment]
> I think it looks good. Do you think the in-memory reader will be sufficient for this, or do we also need to look at #1457?

Good question. I actually forgot to mention that this new feature should by behind a feature flag (default=off). So in the worst case we can just include a note that new feature requires ~4GB of RAM.

> How will we test the final endpoint?

OK, will add a point.

**Comment by bartekn** [comment]
It's live on horizon-testnet:

https://horizon-testnet.stellar.org/accounts?signer=GCJ57EPSX4RX6TPO2QVE7GOJXNGUJARVQYPYFJPFK5Z7SL2USHGNVP2T

Please check it out .
